frontends/ol-search-ui/src/interfaces.ts

```diff
@@ -50,8 +50,9 @@ export type LearningResource = {
   video_id?: string // required for videos
 }
 
-type SearchResult<T> = Omit<T, "topics"> & {
+type SearchResult<T> = Omit<T, "topics" | "lists"> & {
   topics?: string[]
+  lists?: ListItemMember[]
```

Comment:
We should consider changing the `search/` API response to return an empty list for anonymous users, more similar to the other APIs. https://github.com/mitodl/open-discussions/blob/master/search/api.py#L455-L470